xcube/cli/_gen2/genconfig.py

```diff
@@ -207,8 +207,15 @@ def get_schema(cls):
                                             JsonNumberSchema(),
                                             JsonNumberSchema()]),
                 spatial_res=JsonNumberSchema(exclusive_minimum=0.0),
-                time_range=JsonArraySchema(items=[JsonStringSchema(format='date-time'),
-                                                  JsonStringSchema(format='date-time', nullable=True)]),
+                time_range=JsonArraySchema(items=[
```

Comment:
It looks as if we will need this code block in all our store impls and test code. You may create a utility class method in `JsonDatetimeSchema` class:

```python
@classmethod
def new_datetime_range(cls) -> JsonArraySchema: 
    return JsonArraySchema(items=[ ...
```

Reply:
Done. (I used a static method rather than a class method, since it doesn't need access to the class.)